src/lib/spinel/radio_spinel.hpp

```diff
@@ -1286,6 +1287,7 @@ class RadioSpinel : private Logger
     bool mFemLnaGainSet : 1;               ///< Whether FEM LNA gain has been set.
     bool mEnergyScanning : 1;              ///< If fails while scanning, restarts scanning.
     bool mMacFrameCounterSet : 1;          ///< Whether the MAC frame counter has been set.
+    bool mSrcMatchSet : 1;                 ///< Whether the source match feature has been set.
```

Comment:
Shall we add explicit initialization for the two members so that they are set to `0` (in initialization list)?